clarity/contracts/arkadiko-swap-v1-1.clar

```diff
@@ -166,15 +194,22 @@
     )
     (asserts! (and (> x u0) (> new-y u0)) (err ERR-INVALID-LIQUIDITY))
     (asserts! (is-eq swap-token (contract-of swap-token-trait)) (err ERR-WRONG-SWAP-TOKEN))
+    (asserts!
+      (and
+        (is-eq (unwrap-panic (contract-call? .arkadiko-dao get-emergency-shutdown-activated)) false)
+        (is-eq (var-get swap-shutdown-activated) false)
+      )
```

Comment:
```suggestion
      (and
        (not (unwrap-panic (contract-call? .arkadiko-dao get-emergency-shutdown-activated)))
        (not (var-get swap-shutdown-activated))
      )
```
`(not x)` is cheaper to execute than `(is-eq x false)` and I also think it is more readable.